sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -306,6 +306,11 @@ export class AssessmentControllerService {
     // values are used instead to provide the best information as possible.
     let index = 1;
     for (const schedule of assessment.disbursementSchedules) {
+      if (
```

Comment:
We should have the same validation for full-time while reading from the receipt right?? or am I missing something?

Reply:
It does not need to be in the same way because the full-time is based in the receipts. We can add a check before the receipt call (not in the for loop) to avoid one DB query when the disbursement is pending.

Reply:
I added the same test inside the `for` for simplicity.